src/Vocup/MainForm.cs

```diff
@@ -125,9 +125,45 @@ public void UnloadBook(bool fullUnload)
             }
         }
 
+        private void StoreSettings()
+        {
+            //
+            // save the forms location and size
+            //
+            Settings.Default.MainFormLocation = this.Location;
+
+            if (this.WindowState == FormWindowState.Normal)
+            {
+                Settings.Default.MainFormSize = this.Size;
+            }
+            else
+            {
+                Settings.Default.MainFormSize = this.RestoreBounds.Size;
+            }
+            Settings.Default.Save();
+        }
+
+        private void RestoreSettings()
+        {
+            //
+            // Restore the MainForm Location and Size
+            //
+            if (Settings.Default.MainFormLocation != null)
```

Comment:
`System.Drawing.Point` is struct which means it will never be `null`. Unlike below we can't treat a point of zeros as uninitialized because the user can place the window in the upper left corner of the screen which would be point of zeros.